clients/cobol-lsp-vscode-extension/src/__tests__/LanguageClientServiceTest.spec.ts

```diff
@@ -104,4 +104,12 @@ describe("LanguageClientService positive scenario", () => {
             },
         });
     });
+
+    test("Test LanguageClientService exit language server is calls exit on languageClient", () => {
+        jest.clearAllMocks();
+        new JavaCheck().isJavaInstalled = jest.fn().mockResolvedValue(true);
+        vscode.workspace.getConfiguration(expect.any(String)).get = jest.fn().mockReturnValue(9999);
```

Comment:
The test pass even if I remove those three lines. I suggest to remove them.

Reply:
done